t/03-testapi.t

```diff
@@ -21,6 +22,16 @@ use Test::MockModule;
 my $mod       = new Test::MockModule('myjsonrpc');
 my $fake_exit = 0;
 
+# define variables for 'fake_read_json'
+my $report_timeout_called         = 0;
+my $fake_pause_on_timeout         = 0;
+my $fake_needle_found             = 1;
+my $fake_needle_found_after_pause = 0;
+
+# define 'write_with_thumbnail' to fake image
+sub write_with_thumbnail {
```

Comment:
Just an empty sub?

Reply:
It is just to mock that method (in this method we don't care about writing thumbnails).